src/app/components/AllowanceMenu/index.tsx

```diff
@@ -237,7 +240,7 @@ function AllowanceMenu({ allowance, onEdit, onDelete }: Props) {
                 </h2>
                 <div>
                   {permissions.map((permission) => (
-                    <>
+                    <Fragment key={`element-${permission.id}`}>
```

Comment:
This is how I would do it: 
```suggestion
                    <Fragment key={permission.id`}>
```
<Fragment key={permission.id}>